spacialgaze-plugins/SG.js

```diff
@@ -116,6 +116,22 @@ SG.reloadCSS = function () {
 	http.get(options);
 };
 
+//Daily Rewards System for SpacialGaze by Lord Haji
+SG.giveDailyReward = function (userid, user) {
+	if (!user || !userid) return false;
+	userid = toId(userid);
+	if (!user.autoconfirmed) return false;
+	if (!Db('DailyBonus').has(userid)) Db('DailyBonus').set(userid, 1);
+	let seen = Db('seen').get(userid);
+	if ((Date.now() - seen) < 86400000) return false;
+	if ((Date.now() - seen) > 89964000) Db('DailyBonus').set(userid, 1);
+	if (Db('DailyBonus').get(userid) === 8) Db('DailyBonus').set(userid, 1);
+	Db('currency').set(userid, Db('currency').get(userid) + Db('DailyBonus').get(userid));
+	user.send('|popup||wide||html| <center><u><b><font size="3">SpacialGaze Daily Bonus</font></b></u><br>You have been awarded ' + Db('DailyBonus').get(userid) + ' Stardust.<br>' + showDailyRewardAni(userid) + ' <br>Because you are on your ' + Db('DailyBonus').get(userid) + ' Streak.<br>Come Everyday to collect Stardust.(It gets reset every 7 days or if you do not come everyday.)</center>');
```

Comment:
Check your grammar on this.

`Because you are on your 5 Streak.` (example, replace 5 with another number to see more)

This dosen't make sense